src/Term_Command.php

```diff
@@ -3,7 +3,9 @@
 use WP_CLI\Utils;
 
 /**
- * Manage terms.
+ * Manages taxonomy terms and term meta, including create, delete, and list.
```

Comment:
`including` would need to be followed by nouns.

Here's two options:

1. `Manages taxonomy terms and term meta, including creation, deletion, and enumeration`
2. `Manages taxonomy terms and term meta and allows to create, delete, and list them.`

Reply:
Tryig to keep it under 80chars, how about:
`Manages taxonomy terms and term meta, with create, delete, and list commands.`